app/main.py

```diff
@@ -1,13 +1,19 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
```

Comment:
Don't forget that your braces should be located in **one** of two options: open and start with the text or have a line break between the text. You have **both** options here and in the next function 🐞